tests/org/test_add_volunteer_api.py

```diff
@@ -0,0 +1,123 @@
+from org.models import *
+from tests.AuthAPITestCase import AuthAPITestCase
+from rest_framework.test import APITestCase,APIClient
+from rest_framework import status
+from users.models import User
+from org.custom_model_field import PermissionSet as Permissions
+import uuid
+class AddVolunteerAPITestCase(AuthAPITestCase):
+    """
+    This class is to test the API [get] /api/org/[org-id]/volunteer
+    present in the view Org.views.AddVolunteer.get
+    """
+    def setUp(self):
+        """
+            Create an organization in the test database
+        """
+        # Create the org
+        self.org = Org.objects.create(
+            route_slug= 'route_slug',
+            name='test',
+            tagline='test',
+        )
+
+        # Create default group permissions  
```

Comment:
A better way to do the setup will be to call the org API or use the CreateOrgSerializer which has a custom save() has present.

Reply:
I'm having trouble setting this can't we use the existing code for set up?

Reply:
I will provide you better explanation over call